serverless/pages/serverless-differences.asciidoc

```diff
@@ -1,14 +1,108 @@
+
+If you've used {es} before, you'll notice some differences in how you work with the service on serverless, because a number of APIs and settings are not required for serverless projects.
+
+This guide helps you understand what's different, what's available, and how to work effectively when running {es} on {serverless-full}.
+
+[discrete]
+[[elasticsearch-differences-serverless-infrastructure-management]]
+== Fully managed infrastructure
+
+{es-serverless} manages all infrastructure automatically, including:
+
+* Cluster scaling and optimization
+* Node management and allocation
+* Shard distribution and replication
+* Resource utilization and monitoring
+
+This fully managed approach means many traditional {es} infrastructure APIs and settings are not available to end users, as detailed in the following sections.
+
+[discrete]
+[[elasticsearch-differences-serverless-aps-settings]]
+== APIs and settings
+
+Because {es-serverless} manages infrastructure automatically, certain APIs and settings are not available, while others remain fully accessible.
+
+[TIP]
+====
+Refer to the {es-serverless} https://www.elastic.co/docs/api/[API Reference] for a complete list of available APIs.
+====
+
+[discrete] 
+[[elasticsearch-differences-serverless-api-availability]]
+=== API availability
+
+The following categories of operations are *unavailable* since they are managed by the serverless platform:
+
+Infrastructure operations::
+* All `_nodes/*` operations 
+* All `_cluster/*` operations
+* Most `_cat/*` operations, except for index-related operations such as `/_cat/indices` and `/_cat/aliases`
+
+Storage and backup::
+* All `_snapshot/*` operations
+* Repository management operations
 
-preview:[]
+Index management:: 
+* `indices/close` operations
+* `indices/open` operations
+* Recovery and stats operations
+* Force merge operations
+
+[discrete]
+[[elasticsearch-differences-serverless-settings-availability]]  
+=== Settings availability
+
+In {es-serverless}, you can only configure index-level settings.
+Cluster-level settings and node-level settings are not required by end users and the `elasticsearch.yml` file is fully managed by Elastic.
+
+Available settings::
+* *Index-level settings*. Settings that control how {es} documents are processed, stored, and searched are available to end users, including:
+** Analysis configuration
+** Mapping parameters
+** Search/query settings
+** Indexing settings such as `refresh_interval`
+
+Managed settings::
+* *Infrastructure-related settings*. Settings that affect cluster resources or data distribution are not available to end users, including:
+** Node configurations
+** Cluster topology
+** Shard allocation
+** Resource management
+
+[discrete]
+[[elasticsearch-differences-serverless-api-errors]]
+=== API error handling
+
+When attempting to use an unavailable API or setting, you'll receive a clear error message:
```

Comment:
I'd remove `setting` from this (expect there is a different behavior for invalid settings). 

consider parenting this to `API availability` and renaming the section to `APIs` - that way, there's a clear pairing of "here's what's available, and here's what happens when you try to use something unavailable", instead of the settings stuff breaking it up